runtime/common/RunTheKernel.cpp

```diff
@@ -52,12 +52,24 @@ static LayoutInfoType extractLayout(const std::string &kernelName,
   if (!fnOp)
     throw std::runtime_error("Could not find " + kernelName +
                              " function in the module.");
-  if (!fnOp->hasAttr(cudaq::runtime::enableCudaqRun))
-    throw std::runtime_error("Run kernel " + kernelName +
-                             " incorrectly annotated.");
-  auto arrAttr = cast<ArrayAttr>(fnOp->getAttr(cudaq::runtime::enableCudaqRun));
   // Extract layout information from the function's return type.
-  Type returnTy = cast<TypeAttr>(arrAttr[0]).getValue();
+  Type returnTy = [&]() {
```

Comment:
This is patching over a bug. The enableCudaqRun attribute is _required_ to hold the return type now.